fuzz/no_gpu_fuzz/fuzz_targets/curve_bls12_381.rs

```diff
@@ -0,0 +1,101 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use lambdaworks_math::{
+    field::element::FieldElement,
+    cyclic_group::IsGroup,
+    elliptic_curve::{
+        traits::{IsEllipticCurve, IsPairing},
+        short_weierstrass::{
+            curves::bls12_381::{
+                curve::BLS12381Curve, 
+                twist::BLS12381TwistCurve,
+                pairing::BLS12381AtePairing,
+                field_extension::Degree12ExtensionField,
+            },
+            point::ShortWeierstrassProjectivePoint,
+        }
+    },
+    unsigned_integer::element::U384,
+};
+
+type LambdaG1 = ShortWeierstrassProjectivePoint<BLS12381Curve>;
+type LambdaG2 = ShortWeierstrassProjectivePoint<BLS12381TwistCurve>;
+
+//TODO: derive arbitrary for Affine and Projective or change this to use &[u8] as input to cover more cases.
+//TODO: use more advanced options to generate values over curve specifically given most inputs will fail curve check.
+//TODO: Investigate normalization of projective coordinates in arkworks to allow for differential fuzzing.
+fuzz_target!(|values: (u64, u64)| {
+    let (a_val, b_val) = values;
+
+    let a_g1 = BLS12381Curve::generator().operate_with_self(a_val);
+    let b_g1 = BLS12381Curve::generator().operate_with_self(b_val);
+
+    let a_g2 = BLS12381TwistCurve::generator().operate_with_self(a_val);
+    let b_g2 = BLS12381TwistCurve::generator().operate_with_self(b_val);
+
+    // ***AXIOM SOUNDNESS***
+    let g1_zero = LambdaG1::neutral_element();
+
+    let g2_zero = LambdaG2::neutral_element();
+
+    // G1
+    // -O = O
+    assert_eq!(g1_zero.neg(), g1_zero, "Neutral mul element a failed");
+
+    // P * O = O
```

Comment:
It's very uncommon to use multiplicative notation for elliptic curve operations. Please use `+` here and in every other comment.

Reply:
Addressed thanks for the feedback!